salt/_runners/metalk8s_saltutil.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, print_function, unicode_literals
```

Comment:
Is `print_function` used?

Reply:
Maybe not, but it's common practice in Salt modules to always include this very line.